test/test_sparse_ops.py

```diff
@@ -0,0 +1,47 @@
+import math
+from itertools import product, chain
+from numbers import Number
+import random
+
+import unittest
+
+import torch
+
+from torch.testing._internal.common_utils import \
+    (TestCase, run_tests, torch_to_numpy_dtype_dict, suppress_warnings,
+     TEST_NUMPY, IS_MACOS, make_tensor)
+from torch.testing._internal.common_methods_invocations import \
+    (unary_ufuncs)
+from torch.testing._internal.common_device_type import \
+    (instantiate_device_type_tests, ops, dtypes)
+from torch.testing import \
+    (floating_types_and, integral_types, all_types_and_complex_and)
+
+import numpy as np
+
+class TestSparseUnaryUfuncs(TestCase):
+    exact_dtype = True
+
+    @ops((_fn for _fn in unary_ufuncs if _fn.is_sparse_supported is True))
```

Comment:
Create `sparse_ops` using this query and import it, like this:

https://github.com/pytorch/pytorch/blob/a20d4511e40264a900c62ff9631fc063c40ef41e/torch/testing/_internal/common_methods_invocations.py#L861